source/convergence_test.cc

```diff
@@ -434,6 +440,8 @@ void ConvergenceTestData::update_table
 (const DoFHandler<dim, spacedim> &dof_handler,
  const std::map<typename VectorTools::NormType, double> &error_map)
 {
+	dimension = spacedim;
```

Comment:
Dimension has to be always `1` in our case. See [documentation](https://www.dealii.org/current/doxygen/deal.II/classConvergenceTable.html#aa5cfdc644108e52ef46372a2111b9cdc)

Reply:
You are right. Now I understand that there is a difference depending on whether N or h is used.

Fixed in d423619.